ci/authn-azure/run-authn-azure.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# Get an Azure access token
+azure_access_token=$(curl \
+  'http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https%3A%2F%2Fmanagement.azure.com%2F' \
```

Comment:
1. why http and not https ? 
2. why the IP address is not in parameter like CONJUR_SERVER_DNS?

Reply:
This is actually the way to get a managed identity in Azure - https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http.

This URL is accessible to any client application running on the VM that can make HTTP REST calls

Reply:
it is a saved IP in azure, not our env.